modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -66,6 +66,7 @@ function editIssue()
     $fieldsToValidateFirst = array(
```

Comment:
FYI: If your PR addresses a specific redmine ticket, it is always helpful to leave a ticket number in the description, so reviewers and testers can refer to it